util/ranger/points.go

```diff
@@ -339,6 +344,28 @@ func HandlePadCharToFullLength(sc *stmtctx.StatementContext, ft *types.FieldType
 	}
 }
 
+// handleUnsignedIntCol handles negative integer for unsigned integer colunm
```

Comment:
Add some more comments for three returned values.

Reply:
done